script.py

```diff
@@ -0,0 +1,61 @@
+from flask import Flask, request, jsonify
+import mysql.connector
+from dotenv import load_dotenv
+import os
+
+app = Flask(__name__)
+load_dotenv()
+
+def connect_to_db():
+    """
+    A function that creates a connection to db.
+    """
+    return mysql.connector.connect(
+        host=os.getenv("MYSQL_HOST"),
+        user=os.getenv("MYSQL_ROOT_USER"),
+        password=os.getenv("MYSQL_ROOT_PASSWORD"),
+        database=os.getenv("MYSQL_DATABASE"),
+        port=3306
+    )
+
+@app.route('/<table_name>', methods=['POST'])
+def add_data(table_name):
+    """
+    A function that adds a new user to db.
+    """
+    try:
+        mysql_db = connect_to_db() #creates a connection to db
```

Comment:
it's better to connect ot the DB once in the beggining of the app and not per request